### DISCUSSION TITLE
ClickOnce for MAUI Windows Desktop

### DISCUSSION THREAD

**Discussion by cowsen** [dotnet/maui#12153] [Ideas]
Although not cross platform ClickOnce offers a powerful deployment tool for Windows desktop applications with Blazor components to publish to a shared folder with self-signed certificates that a good for a year.  No expensive certificates required!

**Comment by MicroTrendsTom** [comment]
definetly a modern evolution of ClickOnce for MAUI would be perfect 
having spent countless wasted weeks trying to use appstores and deployment and no progress
horrendous time trying  get past the awful legacy of being stuck on WPF.

 MAUI and blazor amazing wonderful and then you try to roll it out... impossible, horrific!
clickonce was a much  better solution  for roll out 

why not to desktop, mobile, tablet, mac, liunux?
multiplatform

**Comment by Xyncgas** [comment]
Maybe simply deploy to app store + executable if possible. On Windows I would like the ability to let people download an exe file to run it directly or an installer that create shortcut on Desktop + StartMenu
